website/content/docs/enterprise/network-segments.mdx

```diff
@@ -11,19 +11,200 @@ description: |-
+    advertise = "{{GetPrivateIP}}"
+    port = 8303
+  },
+  {
+    name = "beta"
+    bind = "{{GetPrivateIP}}"
+    advertise = "{{GetPrivateIP}}"
+    port = 8304
+  }
+]
+```
+
+```json
+{
+  "segments": [
+    {
+      "name": "alpha",
+      "bind": "{{GetPrivateIP}}",
+      "advertise": "{{GetPrivateIP}}",
+      "port": 8303
+    },
+    {
+      "name": "beta",
+      "bind": "{{GetPrivateIP}}",
+      "advertise": "{{GetPrivateIP}}",
+      "port": 8304
+    }
+  ]
+}
+```
+
+</CodeTabs>
+
+The server [agent configuration](/docs/agent/options) options relevant to network
+segments are:
+
+- [`ports.serf_lan`](/docs/agent/options#serf_lan_port): The Serf LAN port on this server
+  for the `<default>` network segment's gossip pool.
+- [`segments`](/docs/agent/options#segments): A list of user-defined network segments
+  on this server, including their names and Serf LAN ports.
+
+## Client Configuration
+
+Each client agent is a member of only one segment at a time. This will be the
+`<default>` segment unless otherwise specified in their [`segment`](/docs/agent/options.html#_segment)
+agent configuration option.
+
+### Join a Client to a Segment ((#join_a_client_to_a_segment))
+
+For a client agent to [join](/commands/join) the Consul
+datacenter, it must connect to another agent (client or server) within its
+configured segment.
+
+<Tabs>
+<Tab heading="Join via another client">
+
+Clients A and B specify the same segment S. Client B is already joined to the segment S
+LAN gossip pool. Client A wants to join via Client B. In order to do so, Client A
+must connect to Client B's configured [Serf LAN port](/docs/agent/options#serf_lan_port).
+
+</Tab>
+<Tab heading="Join via a server">
+
+Client A specifies segment S and wants to join the segment S gossip pool via Server 1.
+In order to do so, Client A must connect to Server 1's configured [Serf LAN port
+for segment S](/docs/agent/options#segment_port).
+
+</Tab>
+</Tabs>
+
+There are several methods to specify the port used by the join operation, listed in order
+of precedence:
+
+1. Specify an explicit port in the join address. For example,
+   `consul agent -retry-join "client-b-address:8303"` when starting Client A using the CLI,
+   or in Client A's configuration using the [retry-join option](/docs/agent/options#retry_join)
+   without cloud auto-join. If using `retry-join`
+   [with cloud auto-join](https://www.consul.io/docs/install/cloud-auto-join#auto-join-with-network-segments),
+   there is no method to specify an explicit port to use, so one of the implicit methods
+   below must be used instead.
+
+2. Specify a Serf LAN port for Client A. Without an explicit Serf LAN port in the join address,
+   Client A will implicitly use its own port number. For example,
+   `consul agent -retry-join "client-b-address" -serf-lan-port 8303` when starting Client A
+   using the CLI, or in Client A's configuration using the
+   [serf_lan](/docs/agent/options#serf_lan_port) option.
```

Comment:
```suggestion
2. Specify an alternate Serf LAN port at the CLI using the `-serf-lan-port` flag (e.g., `consul agent -retry-join "client-b-address" -serf-lan-port 8303`), or in the agent's configuration using the [`ports.serf_lan`] option. When a Serf LAN port is not explicitly specified in the join address, the agent will join the Serf LAN port specified at the CLI or agent configuration.
```

Reply:
Thanks for the cleanup. I made a slightly different set of changes (mostly to the last part). Let me know if you agree:

```
2. Specify an alternate Serf LAN port for the agent. This can be done at the CLI when starting
   the agent (e.g., `consul agent -retry-join "client-b-address" -serf-lan-port 8303`), or in
   the agent's configuration using the [serf_lan](/docs/agent/options#serf_lan_port) option.
   When a Serf LAN port is not explicitly specified in the join address, the agent will use its
   own Serf LAN port number as the join target port. This method requires that the agent and its
   join target use the same Serf LAN port for this segment.
```

Reply:
I find this to be confusing.

> The agent will use its own Serf LAN port number as the join target port. This method requires that the agent and its join target use the same Serf LAN port for this segment.

"Its own port number" implies a local port to me. The phrasing is ambiguous, and could be interpreted that the agent will contact itself when in actuality the agent will initiate a connection a port on a remote host.

This is probably more clear as to the actual operation.

> When a Serf LAN port is not explicitly specified in the join address, the agent will attempt to join the specified host at the Serf LAN port specified in CLI or agent configuration.

Reply:
We don't use the word host anywhere else in the guide. Can I use `target agent` instead of `target host`?

Reply:
Used your wording for the last part of item 2.

Also modified item 3 to include similar text:
```
3. **Use the default Serf LAN port (8301)**. The agent will attempt to join the target host
   on port 8301.
```